src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -856,6 +871,10 @@ def _decode_nonce(backend, nonce):
     OCSPExtensionOID.NONCE: _decode_nonce,
 }
 
+_OCSP_SINGLERESP_EXTENSION_HANDLERS = {
```

Comment:
This doesn't appear to be used, so I'm pretty sure the extension parser below is being created incorrectly